src/java/org/apache/cassandra/locator/AbstractCloudMetadataServiceConnector.java

```diff
@@ -25,22 +25,41 @@
 import java.net.URL;
 import java.util.Map;
 
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableMap;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 abstract class AbstractCloudMetadataServiceConnector
 {
+    static final String METADATA_URL_PROPERTY = "metadata_url";
```

Comment:
CassandraRelevantProperties?

Reply:
I dont think that is actually necessary. It is not like this property is a _system property_. This is just a key of a property in cassandra-rackdc.properties.

Reply:
ah, yes, right